blogposts/2021/dev-tool-time-primeagen.md

```diff
@@ -0,0 +1,104 @@
+
+<figure>
+    <div class="container my-4 video-embed embed-responsive embed-responsive-16by9">
+        <video controls src="https://sourcegraphstatic.com/blog/dev-tool-time/primeagen/thinking.mp4"></video>
+    </div>
+    <figcaption style="text-align: center; font-style: italic;">Listen to ThePrimeagen discuss the importance of reducing complexity to reduce unnecessary thinking</figcaption>
+</figure>
+
+## `git worktree` so you don't have to `git stash`
+
+Reducing cognitive overhead is as much about taking advantage of existing tools and workflows as it is extensive configuration. One of ThePrimagen's examples of this is his relatively recent switch to using Git worktrees, which allow you to concurrently checkout multiple branches of a repository. Rather than going through the steps of stashing changes, switching branches, and later on figuring out what to pop, worktrees are an interface to a simpler model of navigating branches.
+
+<figure>
+    <div class="container my-4 video-embed embed-responsive embed-responsive-16by9">
+        <video controls src="https://sourcegraphstatic.com/blog/dev-tool-time/primeagen/worktree.mp4"></video>
+    </div>
+    <figcaption style="text-align: center; font-style: italic;">Watch ThePrimeagen demonstrate one of his favorite Git features: worktrees</figcaption>
+</figure>
+
+## Even milliseconds matter (to ThePrimeagen)
+
+In his VimConf 2020 talk, ["Vim Speed and Excellence"](https://www.youtube.com/watch?v=tCktGgPQ3D0&list=PLcTu2VkAIIWzD2kicFNHN2c35XQCeZdsv), ThePrimeagen examines the ways that using particular commands in Vim can be faster or slower for himself, and, by extension, for other individuals. It comes down to milliseconds of gain, but that's worth it for someone who is perpetually evolving a development environment for speed, efficiency, and, unobtrusiveness. ThePrimeagen thinks a lot about his setup sometimes, so that the rest of the time, when he's using it, he doesn't really have to think about it at all. Instead, he just codes.
+
+<figure>
+    <div class="container my-4 video-embed embed-responsive embed-responsive-16by9">
+        <video controls src="https://sourcegraphstatic.com/blog/dev-tool-time/primeagen/milliseconds.mp4"></video>
+    </div>
+    <figcaption style="text-align: center; font-style: italic;">Listen to ThePrimeagen's deep dive on Vim keybindings and time</figcaption>
+</figure>
+
+## Show notes
+
+- [AKG K712 headphones](https://www.akg.com/Headphones/Professional%20Headphones/K712PRO.html?dwvar_K712PRO_color=Black-GLOBAL-Current&cgid=Professional%20Headphones)
+- [Acer Predator monitor](https://www.acer.com/ac/en/US/content/predator-models/monitors)
+- [Ansible](https://www.ansible.com/)
+- [Audio Technica microphones](https://www.audio-technica.com/en-us/microphones/wired)
+- [Dvorak keyboard layout](https://en.wikipedia.org/wiki/Dvorak_keyboard_layout)
+- [Elgato capture card](https://www.elgato.com/en/game-capture-4k60-s-plus) - is this the right one?
```

Comment:
Just making a note here to make sure this text is edited once we figure out if it's the right one. Feel free to resolve if you're got it :)